LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -487,4 +488,4 @@
     "web-platform-testscss/css-transforms/": "import",
     "web-platform/test/html/infrastructure": "import",
     "webrtc-platform-tests/webrtc": "import"
-}
\ No newline at end of file
+}
```

Comment:
Including the text-indent directory isn't particularly useful when its parent directory is included.

Can you undo the changes in this file?

`git checkout HEAD^ -- LayoutTests/imported/w3c/resources/import-expectations.json` should do it.